stl/inc/type_traits

```diff
@@ -1368,45 +1368,56 @@ struct _Common_reference2B<_Ty1, _Ty2, void_t<_Basic_specialization<_Ty1, _Ty2>>
     using type = _Basic_specialization<_Ty1, _Ty2>;
 };
 
-// N4917 21.3.8.7 [meta.trans.other]/5.3.1: "If T1 and T2 are reference types and COMMON_REF(T1, T2) is well-formed..."
+// N4928 [meta.trans.other]/5.3.1: "If T1 and T2 are reference types and COMMON_REF(T1, T2) is well-formed..."

```

Comment:
This is the paragraph that's being updated by WG21-P2655R3. It's a bit awkward since we don't have the new WP yet, but I think we should cite the updated wording here (and note with a TRANSITION that we need a new WP number).

This also occurs in `VSO_0000000_type_traits\test.cpp` right line 1366.